woo-razorpay.php

```diff
@@ -257,7 +295,102 @@ public function init_form_fields()
                     'description' => __('Webhook secret is used for webhook signature verification. This has to match the one added <a href="https://dashboard.razorpay.com/#/app/webhooks">here</a>', $this->id),
                     'default' => ''
                 ),
-
+                'enable_1cc' => array(
```

Comment:
@ChetanGN , Please manage these from module file. Take the reference from `route` module.